packages/ozone/src/db/migrations/20250618T150931000Z-update-appeal-reason-stats.ts

```diff
@@ -0,0 +1,309 @@
+import { Kysely, sql } from 'kysely'
+import { OZONE_APPEAL_REASON_TYPE } from '../../api/util'
+import { REASONAPPEAL } from '../../lexicon/types/com/atproto/moderation/defs'
+import { DatabaseSchemaType } from '../schema'
+import * as modEvent from '../schema/moderation_event'
+import * as recordEventsStats from '../schema/record_events_stats'
+
+export async function up(db: Kysely<any>): Promise<void> {
```

Comment:
When I read through this I was trying to identify the "diff" from the current materialized view, I think we could leave some comments that make it clearer what the specific differences are.